draft-ietf-cats-framework.md

```diff
@@ -186,7 +192,7 @@ Service request:
  : Service requests are not explicitly sent by clients to CATS-Forwarders.
 
 CATS-Forwarder:
- : A network entity that makes forwarding decisions based on CATS information to steer traffic specific to a  service request towards a corresponding yet selected service contact instance. The selection of a service contact instance relies upon a multi-metric path computation.
+ : A network entity that steers traffic specific to a service request towards a corresponding yet selected service contact instance according to the forwarding decisions made by the decision point based on CATS information. The decision point, which is on the C-PS, is responsible for making the steering policy, and it may or may not on the CATS-Forwarder.
```

Comment:
```suggestion
 : A network entity that steers traffic specific to a service request towards a corresponding yet selected service contact instance according to provisioned forwarding decisions. These decisions are supplied by a C-PS, which may or may not be on the CATS-Forwarder.
```

Reply:
The tweaked text seems to conflict with this part "the choice of the service contact instance may be left open to the Egress CATS-Forwarder" in subsequent part of the text.